features/support/env.rb

```diff
@@ -139,3 +143,9 @@ def extract_process_id(data)
   stdout = aruba.command_monitor.find(Aruba.platform.detect_ruby(last_command_started)).send(:stdout)
   expect(@service_current_process_id).not_to eq(extract_process_id(stdout))
 end
+
+Then(/^the binary for "([^"]*)" should be installed$/) do
```

Comment:
Not sure whether you got the comment, but hard coding the version is unfortunate here.